### DISCUSSION TITLE
Problem connecting my users.

### DISCUSSION THREAD

**Discussion by Mrsuupra** [games647/FastLogin#1136] [Q&A]
It doesn't let some of my users connect and this will always appear in the console. 
How can I let them enter without a problem (it's not premium)
![image](https://github.com/games647/FastLogin/assets/155863407/bac3bf97-4c81-4af6-9dc2-4dbe25387fd4)

**Comment by TuxCoding** [comment]
Disable auto register. 

Considering player is not premium, fast login shouldn't request an online mode authentication which it does in your log.
